src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -183,11 +183,11 @@ protected void actionPerformed( final GuiButton btn )
 
 				if( btn == this.terminalStyleBox )
 				{
-					AEConfig.instance.settings.putSetting( iBtn.getSetting(), next );
+					AEConfig.instance().getConfigManager().putSetting( iBtn.getSetting(), next );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.client.gui.implementations.GuiMEMonitorable.actionPerformed(GuiButton) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)